tests/php_hobby/test.sh

```diff
@@ -0,0 +1,192 @@
+GREEN='\033[0;32m'
+
+total_tests=6 # only 7 tests are currently being done, this value should be changed if tests increases
+
+echo -n "Enter First Name : "
+read first_name
+
+echo -n "Enter Last Name : "
+read last_name
+
+set_folder() {
+    first_name=$(echo "$first_name" | tr '[:upper:]' '[:lower:]')
+    first_letter=$(echo "${last_name:0:1}" | tr '[:lower:]' '[:upper:]')
+    last_substring=$(echo "${last_name:1}" | tr '[:upper:]' '[:lower:]')
+
+    folder_name="$first_name$first_letter$last_substring"
+}
+
+set_folder
+
+submission_directory="$PWD/../../submissions"
+path_to_folder="$submission_directory/$folder_name"
+path_to_php_folder="$path_to_folder/php"
+path_to_log="$submission_directory/../tests/logfile.json"
+
+check_php_folder_existence() {
+    folder_count=$(ls $path_to_folder | grep "php" -c)
+    if [ $folder_count -eq 0 ]
+    then
+        php_folder_exists=0
+        echo ""
+        echo -e "${BOLD}Checking the php folder ...${NONE}"
+        echo -e "${RED}Your folder should be named ${BOLD}\"php\"${NONE} ${RED}and located in the folder with your First and Last name. (${BOLD}\"$folder_name/database\"${NONE}${RED})${NONE}"
+    else
+        php_folder_exists=1
+    fi
+}
+
+
+check_file_content1() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep  "<?php"  $currentFile  && grep "echo" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo ""
+            echo -e "1. Ensure your file contains php syntax"${NONE}
+        fi
+   done
+}
+
+check_file_content2() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep  "["  $currentFile && grep "]" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo -e "${BOLD}Checking the php file ...${NONE}"
+            echo -e "1. Ensure your code contains array(s)"${NONE}
+        fi
+   done
+}
+
+check_file_content3() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep  "i++"  $currentFile && grep "0" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo -e "${BOLD}Checking the php file ...${NONE}"
+            echo -e "1. Ensure you loop through the array"${NONE}
+        fi
+   done
+}
+
+check_file_content4() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep "="  $currentFile && grep "," $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo -e "${BOLD}Checking the php file ...${NONE}"
+            echo -e "1. Ensure you loop through the array(s)"${NONE}
+        fi
+   done
+}
+
+check_file_content5() {    
+    for currentFile in $path_to_php_folder/*
+    do
+        if grep "["  $currentFile && grep "]" $currentFile
```

Comment:
You doing this check twice??

Reply:
No, I was checking the opening and closing brackets of the array.

Reply:
@Ireti203 check_file_content2 and check_file_content5() are the same thing now.